schema/functions.json

```diff
@@ -90,6 +90,25 @@
         "name",
         "operation"
       ]
+    },
+    "operation": {
+      "oneOf": [
+        {
+          "type": "string",
+          "description": "If type is `openapi`, <path_to_openapi_definition>#<operation_id>. If type is `asyncapi`, <path_to_asyncapi_definition>#<operation_id>. If type is `rpc`, <path_to_grpc_proto_file>#<service_name>#<service_method>. If type is `graphql`, <url_to_graphql_endpoint>#<literal \\\"mutation\\\" or \\\"query\\\">#<query_or_mutation_name>. If type is `odata`, <URI_to_odata_service>#<Entity_Set_Name>. If type is `expression`, defines the workflow expression.",
+          "minLength": 1
+        },
+        {
+          "type": "object",
+          "description": "OpenAPI Path Object definition",
```

Comment:
@ricardozanini @shaisachs makes a good (second) point. WDYT?

Reply:
We discussed the first point offline and I get it now so no worries on that point. The second point is much more tactical and I think still something to consider.

Reply:
@ricardozanini Any thoughts/feedback?

Reply:
Let's move to `Path Item`!!

Many thanks, @shaisachs! @cdavernas I'll review it today so we can move forward.

Reply:
@shaisachs @cdavernas [I don't see the `path` attribute in the `path item` object](https://spec.openapis.org/oas/v3.1.0#path-item-object-example). We need the path, otherwise, we can't determine which path/function to call. That said, we should follow the Paths Object instead exactly as stated in this PR.